pennychat/serializers.py

```diff
@@ -71,3 +71,9 @@ class FollowUpSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = FollowUp
         fields = ['id', 'url', 'penny_chat', 'content', 'date', 'user']
+
+
+class FollowUpWriteSerializer(serializers.ModelSerializer):
```

Comment:
I used the a mode serializer for a write serializer because the process of saving information to the database shouldn't be concerned with what output we choose to generate for any view